grails-app/migrations/0.8.x/changelog-2022-09-02-1200-add-default-product-type.xml

```diff
@@ -31,4 +31,20 @@
       UPDATE product SET product_type_id = (SELECT id FROM product_type WHERE id like "DEFAULT" OR code like "DEFAULT" OR name like "DEFAULT" LIMIT 1) WHERE product_type_id IS NULL;
     </sql>
   </changeSet>
+  <changeSet author="awalkowiak" id="0209202212000-2">
+    <preConditions onFail="MARK_RAN">
+      <and>
+        <sqlCheck expectedResult="1">
+          SELECT COUNT(*) FROM product_type WHERE id="DEFAULT"
```

Comment:
Please make make the check here as it was done before, which is:
`<sqlCheck expectedResult="0">SELECT COUNT(*) FROM product_type WHERE id like "DEFAULT" OR code like "DEFAULT" OR name like "DEFAULT"</sqlCheck>`